yarn.lock

```diff
@@ -7364,7 +7364,7 @@ aws4@^1.8.0:
   resolved "https://registry.yarnpkg.com/aws4/-/aws4-1.8.0.tgz#f0e003d9ca9e7f59c7a508945d7b2ef9a04a542f"
   integrity sha512-ReZxvNHIOv88FlT7rxcXIIC0fPt4KZqZbOlivyWtXLt8ESx84zd3kMC6iK5jVeS2qt+g7ftS7ye4fi06X5rtRQ==
 
-axe-core@4.4.1, axe-core@4.4.3, axe-core@^3.5.5, axe-core@^4.0.2, axe-core@^4.2.0:
+axe-core@4.4.3, axe-core@4.7.2, axe-core@^3.5.5, axe-core@^4.0.2, axe-core@^4.2.0:
```

Comment:
I see multiple axe-core versions.
Can you run 'yarn dedupe' to resolve these.

Reply:
Ok

Reply:
`yarn run dedupe`

Reply:
No changes to the lockfile were made on running this
<img width="256" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/0dab6a15-1741-477b-8a58-1cc0769da2bf">
Is it possible that the certain packages have their version's locked?

Reply:
<img width="218" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/ccbb734b-3e1e-4e9f-9e52-aa6c93619d9b">
Here's one set to 3.5.5

Reply:
Looks like your changes are not complete. Refer this PR where the version was updated earlier https://github.com/microsoft/fluentui/pull/20609

Reply:
<img width="219" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/a0b0c29c-e2c2-4129-bef6-1861c19a4c0b">
<img width="211" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/887eaad2-e38e-4176-b7a8-7f21bf25dbf6">

Updated all instances of jest-axe and axe-core.
Let me build and run the tests to see if anything breaks.